### ISSUE TITLE
Either wrong port and ~15 peers or correct port and 1-3 peers

### ISSUE THREAD

**Issue by BoBeR182** [OpenBazaar/OpenBazaar#1198]
With no arguments I get about 15 peers but https://tools.bizarre.company/port-check/ fails

```
 ./openbazaar start
bober@BoberBot:~/OpenBazaar$Checking NAT Status...
{'nat_type': 'Restric NAT', 'external_ip': 'REDACTED', 'external_port': 34366}
Setting up UPnP Port Map Entry...
UPnP TCP P2P Port Map configuration done  (34366 -> 34366) => True
UPnP UDP P2P Port Map configuration done  (34366 -> 34366) => True
Started OpenBazaar Web App at http://127.0.0.1:32924
(process:30204): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed
```
The code below gets me 3 nodes max but causes https://tools.bizarre.company/port-check/ to pass

```
 ./openbazaar --server-port 12./openbazaar --server-ip REDACTED --server-port 12345 --disable-stun-check start 
bober@BoberBot:~/OpenBazaar$ Setting up UPnP Port Map Entry...
UPnP TCP P2P Port Map configuration done  (12345 -> 12345) => True
UPnP UDP P2P Port Map configuration done  (12345 -> 12345) => True
Started OpenBazaar Web App at http://127.0.0.1:52238
(process:29231): GLib-CRITICAL **: g_slice_set_config: assertion 'sys_page_size == 0' failed
```

**Comment by hoffmabc** [comment]
Have you tried this on the latest (0.4) build?

**Comment by BoBeR182** [comment]
Consider closed as of build 0.4
